dev/src/dev/render_png.clj

```diff
@@ -85,29 +90,123 @@
     (.deleteOnExit tmp-file)
     (open tmp-file)))
 
-(comment
-  (render-card-to-png 1)
-
-  (let [{:keys [content]} (render-pulse-card 1)]
-    (open-hiccup-as-html content))
-
-  ;; open viz in your browser
-  (-> [["A" "B"]
-       [1 2]
-       [30 20]]
-      (render.tu/make-viz-data :line {:goal-line {:graph.goal_label "Target"
-                                                  :graph.goal_value 20}})
-      :viz-tree
-      open-hiccup-as-html)
-
-  (-> [["As" "Bs" "Cs" "Ds" "Es"]
-       ["aa" "bb" "cc" "dd" "ee"]
-       ["aaa" "bbb" "ccc" "ddd" "eee"]]
-      (render.tu/make-viz-data :table {:reordered-columns   {:order [2 3 1 0 4]}
-                                       :custom-column-names {:names ["-A-" "-B-" "-C-" "-D-"]}
-                                       :hidden-columns      {:hide [0 2]}})
-      :viz-tree
-      open-hiccup-as-html))
+(def ^:private execute-dashboard #'pulse/execute-dashboard)
```

Comment:
I usually just call private functions as regular aliased functions -- `(#'pulse/execute-dashboard ...)` inline. Were you just defining this here for convenience?